### ISSUE TITLE
Fix Travisci.yml

### ISSUE THREAD

**Issue by euventura** [convenia/checklistable#4]
fix travis yam to build project with success

**Comment by lukbukkit** [comment]
Hello,
I would like to help you. If it's okay, then I would try to complete this task 😉

**Comment by edbizarro** [comment]
hi @LukBukkit 

Sure! glad u could help

**Comment by lukbukkit** [comment]
Okay let the battle with Travis begin 😅

**Comment by lukbukkit** [comment]
Here are my first results:
The builds failed, because Composer couldn't resolve the `dependencies through SAT` (this problem also exists on CircleCI). For a example see [here](https://travis-ci.org/LukBukkit/checklistable/jobs/289565983). I tried some things and came to the conclusion that if we drop the pipes within the version constants it'll work. 
(As an example: `"illuminate/support": "~5.2|~5.3|~5.4|~5.5"` => `"illuminate/support": "~5.2"`)
Now it's up to you what we should do with this. But I've got an idea: If you want to support multiple versions, you could also use just the lowest version of the library.

**Comment by euventura** [comment]
did you try with 5.* ?

**Comment by edbizarro** [comment]
using the lowest version in composer.json will resolve correctly for newer versions too?

**Comment by lukbukkit** [comment]
@edbizarro Yeah it should work if you use `~5.0`: [Travis](https://travis-ci.org/LukBukkit/checklistable/jobs/289632532) - [Packagist Semver Checker](https://semver.mwl.be/#?package=illuminate%2Fsupport&version=~5.0&minimum-stability=stable)
But it didn't seem to work with `~5.2`: [Travis](https://travis-ci.org/LukBukkit/checklistable/builds/289636542)

@euventura It works also if we use `5.*`: [Travis](https://travis-ci.org/LukBukkit/checklistable/jobs/289634429) - [Packagist Semver Checker](https://semver.mwl.be/#?package=illuminate%2Fsupport&version=5.*&minimum-stability=stable)

**Comment by lukbukkit** [comment]
But on CircleCI the [results](https://circleci.com/gh/LukBukkit/checklistable) look different. Maybe it's caused by other machine specs.

**Comment by euventura** [comment]
maybe we need to use 5.*

**Comment by lukbukkit** [comment]
Yeah that would be a option but maybe we can look for other libraries and how they define their dependencies first.
